coverage/upload-to-coveralls.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# shellcheck source=coverage-utils.sh
+source "${SMALLTALK_CI_BUILD}/coverage-utils.sh"
+
+readonly COVERALLS_API="https://coveralls.io/api/v1/jobs"
+readonly coverage_results="${SMALLTALK_CI_BUILD}/coveralls_results.json"
```

Comment:
`coverage_results` definitions are inconsistent, sometimes with `readonly` sometimes not. Let's use `readonly COVERAGE_RESULTS`, it's a constant, right?